src/main/java/ai/deepcode/javaclient/core/DeepCodeParamsBase.java

```diff
@@ -91,12 +91,16 @@ public void setApiUrl(@NotNull String apiUrl) {
   }
 
   public void setApiUrl(@NotNull String apiUrl, boolean disableSslVerification) {
+    setApiUrl(apiUrl, disableSslVerification, false);
+  }
+
+  public void setApiUrl(@NotNull String apiUrl, boolean disableSslVerification, boolean requestLogging) {
     if (apiUrl.isEmpty()) apiUrl = "https://www.deepcode.ai/";
```

Comment:
Should we throw instead? This url is probably not valid any more for tests

Reply:
or use our deeproxy URL by default?

Reply:
I'll change it to the deeproxy url